Api/Test/SectionService.test.ts

```diff
@@ -0,0 +1,101 @@
+import * as mongoose from 'mongoose';
+import SectionRepository from '../Src/Repositories/SectionRepository';
+import SectionService from '../Src/Services/SectionService';
+import SectionSchema from '../Src/Models/Section';
+import { Section } from '../Src/Models/Section';
+
+type SectionDBModel = Section & {_id: mongoose.Types.ObjectId};
+
+class TestSectionRepository implements SectionRepository {
+    private sections: Array<SectionDBModel> = [];
+    model: any;
+
+    async getAll(){
+        return this.sections;
+    };
+
+    async getById(id: mongoose.Types.ObjectId){
+        return this.sections.find(project => project._id === id);
```

Comment:
😳 this is literally the only thing i found, should be section here not project 🙈

Reply:
copying went wrong :D fixed, thanks for noticing